src/modules/tas_studio/mod.rs

```diff
@@ -433,6 +436,71 @@ fn replay(marker: MainThreadMarker) {
     };
 }
 
+static BXT_TAS_STUDIO_REPLAY_VIEWS: Command = Command::new(
+    b"bxt_tas_studio_replay_views\0",
+    handler!(
+        "bxt_tas_studio_replay_views
+
+Replays the currently loaded camera views of TAS up to the stop frame.",
+        replay_views as fn(_)
+    ),
+);
+
+fn replay_views(marker: MainThreadMarker) {
+    let mut state = STATE.borrow_mut(marker);
+    *state = match mem::take(&mut *state) {
+        State::Editing {
+            mut editor,
+            last_generation,
+            last_branch_idx,
+            simulate_at,
+            bridge,
+        } => {
+            editor.cancel_ongoing_adjustments();
+
+            sdl::set_relative_mouse_mode(marker, true);
+            client::activate_mouse(marker, true);
+
+            // bxt_tas_norefresh_until_last_frames = 0 means 1 frame being played. Heh.
+            let start_frame = norefresh_until_stop_frame_frame_idx(marker, &editor);
+
+            engine::prepend_command(marker, "bxt_hud 0; hud_draw 0\n");
```

Comment:
Do we want this?

Reply:
It was fairly subjective because I did make it that way. My reasoning is that since we only change the player's views, none of the stats actually get changed, so none of the HUD info would matter.